### DISCUSSION TITLE
Commenting badge for org members

### DISCUSSION THREAD

**Discussion by tramcrazy** [forem/forem#17201] [Icebox]
It would be great if members of the organisation who posted something on Forem had a badge like the Author badge in the comment section. I work with others on our project and different ones of us blog about it, but it would be helpful for people to see at a glance that it's someone from the organisation who are responding to them.

**Comment by James-Livesey** [comment]
Yeah, for example @tramcrazy and I are both part of an org, and it would be nice if whenever I post a comment on his posts for the org and vice versa, we have some sort of **Representative for XYZ** or **Org member** badge or something!

That way, other commentors can easily distinguish official replies from an org by looking at who wrote the reply.

**Comment by packetspoofer** [comment]


**Reply by James-Livesey** [reply]
Well, our org is [@liveg](https://dev.to/liveg) if that's what you mean, but ideally I would think that a lot of other orgs with a few active members in them would benefit from having org member badges when commenting in posts that are part of the org they're in.

**Comment by JennieOcken** [comment]
Thank you all for this great discussion.  This kind of flair is something we have definitely discussed before.  In fact we have an [issue here](https://github.com/forem/forem/issues/9620) for exactly this idea (with some slight variations).  I believe we will get to this but don't have it in our immediate roadmap.  So I am going to put this in our "icebox".  What that means is I think it's a great idea just not part of our immediate priorities.

Thanks!
